modules/nf-core/comebin/runcomebin/tests/main.nf.test

```diff
@@ -0,0 +1,111 @@
+nextflow_process {
+
+    name "Test Process COMEBIN_RUNCOMEBIN"
+    script "../main.nf"
+    process "COMEBIN_RUNCOMEBIN"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "comebin"
+    tag "comebin/runcomebin"
+
+    // setup {
+
+    //     run("MINIMAP2_ALIGN") {
```

Comment:
I would remove this entirely and leave as just a comment saying requires too big data thus only stub test  - people may want to try and uncomment and use the sanger test data, but nf-core can't guarantee stability of this